app/models/container.rb

```diff
@@ -4,6 +4,7 @@ class Container < ActiveRecord::Base
 
   has_one    :container_group, :through => :container_definition
   delegate   :ext_management_system, :to => :container_group
+  delegate   :container_replicator, :to => :container_group
```

Comment:
@blomquisg yes I was questioning the use of `delegate` vs `:through`, e.g.:

    has_one :container_replicator, :through => :container_group

It should work right? (preferred too?)